tests/run.sh

```diff
@@ -60,6 +60,8 @@ stop_services() {
 
 start_upstream_tidb() {
     cat - > "$OUT_DIR/tidb-config.toml" <<EOF
+new_collations_enabled_on_first_bootstrap = true
```

Comment:
it should still work if new_collations is disabled right? although now we run all test enabling it

Reply:
yes, it works for old collations, but new_collations code path seems to be more complex and more valuable to test(it's enabled by default in tidb-operator or tidb-cloud)